Description: Change the type of documentation builder to text
 Because of problems with external references and tracking pixels,
 switch the type of documentation build to plain text.
Author: Harlan Lieberman-Berg <H.LiebermanBerg@gmail.com>
Forwarded: not-needed
--- a/docsite/build-site.py
+++ b/docsite/build-site.py
@@ -43,7 +43,7 @@
         print "Creating html documentation ..."
 
         try:
-            buildername = 'html'
+            buildername = 'text'
 
             outdir = os.path.abspath(os.path.join(os.getcwd(), "htmlout"))
             # Create the output directory if it doesn't exist
